src/Views/ShortcutsView.vala

```diff
@@ -77,14 +114,16 @@ public class ShortcutOverlay.ShortcutsView : Gtk.Grid {
         column_end.attach (new ShortcutLabel.from_gsettings (SCHEMA_MEDIA, "screensaver"), 1, 5);
         column_end.attach (new NameLabel (_("Log out:")), 0, 6);
         column_end.attach (new ShortcutLabel.from_gsettings (SCHEMA_MEDIA, "logout"), 1, 6);
+        column_end.attach (new NameLabel (_("Switch layout:")), 0, 7);
```

Comment:
Since we don't have the context of "keyboard" anywhere here (like we do in System Settings → Keyboard), I wonder if we should do:

```suggestion
        column_end.attach (new NameLabel (_("Switch keyboard layout:")), 0, 7);
```